README.md

```diff
@@ -66,39 +66,42 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
     ./install.sh
     ```
 
-4. Apache configuration and restart 
-LORIS requires Apache's mod_rewrite module to rewrite its URLs. Enable this module, then restart Apache: 
-
+4. Run the makefile (use `make dev` if you are setting up a development sandbox)
     ```
-    sudo a2enmod rewrite
-    sudo service apache2 reload
+    cd /var/www/$projectname
+    make
     ```
 
-5. Go to http://localhost/installdb.php and follow the instructions to finalize LORIS installation.
-
-    _Note_: Apache config files will be installed as *.conf, per Ubuntu 14.04. If running an earlier version of Ubuntu, rename these files, then run the following commands. After, restart Apache.
+5. Apache configuration
 
+If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:
 
     ```
+    sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
+    sudo service apache2 reload
```

Comment:
```suggestion
    sudo systemctl reload apache2
```
New way with systemd,  old way still work

Reply:
@PapillonMcGill 
In the other Readme files and wikis, We always use the "sudo service apache2 reload". If you prefer the new way, we could change it in the other PR to change all of them.

Reply:
Yes, we can do it in an separate PR, to have the same every where.